cmd/apiserver/main.go

```diff
@@ -34,11 +38,8 @@ import (
 	"github.com/google/exposure-notifications-verification-server/pkg/ratelimit"
 	"github.com/google/exposure-notifications-verification-server/pkg/ratelimit/limitware"
 	"github.com/google/exposure-notifications-verification-server/pkg/render"
-
-	"github.com/google/exposure-notifications-server/pkg/keys"
-	"github.com/google/exposure-notifications-server/pkg/logging"
-	"github.com/google/exposure-notifications-server/pkg/observability"
-	"github.com/google/exposure-notifications-server/pkg/server"
+	"github.com/opencensus-integrations/redigo/redis"
+	"go.opencensus.io/stats/view"
```

Comment:
Move these to the last import block

Reply:
Done, refactored to proper import block and then sorted the imports